pkg/operator/detectinputchange/as_configmap.go

```diff
@@ -0,0 +1,115 @@
+package detectinputchange
+
+import (
+	"encoding/base64"
+	"fmt"
+	"hash/fnv"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes"
+)
+
+func GetConfigMapHash(obj *corev1.ConfigMap) (string, error) {
```

Comment:
godocs

Reply:
also looks like something that should live in library-go